modules/mod_admin/templates/_admin_view_types.tpl

```diff
@@ -0,0 +1,7 @@
+{% block content_types %}
+    {% for content_type, url in id|content_type_urls %}
```

Comment:
Maybe put the text/html version on top and suppress it in the loop below?  Then we can also give the top one an understandable title like @ArthurClemens suggested: "View as rendered page" (or something similar)